src/lib/modernExtend.ts

```diff
@@ -1894,15 +1894,18 @@ function genericMeter(args?: MeterArgs) {
         if (args.producedEnergy !== false) exposes.push(e.produced_energy().withAccess(ea.STATE_GET));
         fromZigbee = [args.fzElectricalMeasurement ?? fz.electrical_measurement, args.fzMetering ?? fz.metering];
         toZigbee = [
```

Comment:
```suggestion
        const useMetering = args.power !== false && args.power?.cluster === "metering";
        toZigbee = [
```

Then use it below (to avoid having the same condition twice.